modules/ROOT/pages/microsoft-azure-sb-connector.adoc

```diff
@@ -0,0 +1,334 @@
+
+When you create the app:
+
+* Set the *Unix CRON Expression* field to a CRON expression that follows the UNIX standard. For example, to connect and receive messages every day at 8 AM, use `0 8 * * *`.
+* Set the *Max. Messages to Receive* field to the maximum number of messages for the connector to receive every time the CRON expression triggers a receive.
+
+The connector executes a batch receive every time the CRON expression commands it. Any number of available messages up to the maximum set will be retrieved and processed by the flow.
+
+[[batch]]
+=== Queue Send Batch
+
+The following example configures a Mule app to send multiple messages to a queue:
+
+. Create a Mule app to receive messages from a queue or topic, as described in <<basic>>.
+. Replace the call to the *Publish message* operation with a call to the *Publish batch of messages* Operation. Make sure the *Destination type* and *Destination name* are properly set.
+. When crafting the message, be sure to create a list of strings, since the app will send many messages simultaneously.
++
+*Reviewers: what is the format of the list of strings? Is it comma-separated?*
++
+To split a string by the commas, and send every piece as an individual message, drag a *Transform message* and use the following script:
+```
+%dw 1.0
+%output application/java
+---
+payload splitBy ','
+```
+
+[[enqueuing]]
+=== Schedule the Logical Enqueuing of a Message
+
+The following example configures a Mule app to send a message to all listeners at a specified time:
+
+. Create a Mule app to receive messages from a queue or topic, as described in <<basic>>.
+. Fill in the "Scheduled enqueue time UTC" of the *Publish message* call parameter with the date/time you want the message to be dispatched. For example: `2019-06-27T21:16:46.866Z`.
+
+*Reviewers, where is the Publish message call parameter? I am not seeing it.*
+
+The message will be sent to the destination, which will dispatch the message at the specified time.
+
+== REST operations
+
+* Queue
+** Create
+** Get
+** Get Size
+** List
+** Update
+** Delete
+* Topic
+** Create
+** Get
+** List
+** Update
+** Delete
+* Subscription
+** Create
+** Get
+** Get Size
+** List
+** Update
+** Delete
+* Rule
+** Create
+** Get
+** List
+** Update
+** Delete
+
+== Additional Configuration Information
+
+=== Queue and Subscription Listener Operations
+
+The output payload is a string with the message. The connector returns additional attributes in the outbound properties.
+
+=== Restricted Access Policies
+
+In situations where you hour resources can only send and receive amqp messages, enable the option *AMQP-only credentials* inside the *Advanced* tab.
+
+=== Handling a Very High Number of Messages
+
+When handling a very high number of messages (such as thousands per second), especially if publishing to and reading from the same destination in the same Mule app, you might see messages like the following in the log:
```

Comment:
ditto prev. comment about "handling"

"If your <blah> processes a very high number of messages (such as thousands per second), especially if it publishes to and reads from the same destination in one Mule app, you might see...."